RxSwift/Observables/Observable+Single.swift

```diff
@@ -196,3 +196,17 @@ extension ObservableType {
         return Scan(source: self.asObservable(), seed: seed, accumulator: accumulator)
     }
 }
+
+// MARK: defaultIfEmpty
+
+extension ObservableType {
+    /**
+    Emit items from the source Observable, or a default item if the source Observable emits nothing
+     
+     - seealso: [DefaultIfEmpty operator on reactivex.io](http://reactivex.io/documentation/operators/defaultifempty.html)
+     - parameter defautValue: An default value to be sent if the source does not emit any elements
+    */
+    public func defaultIfEmpty(_ defautValue: E) -> Observable<E> {
```

Comment:
`defautValue` :)

I'm thinking that the name would probably be more in the spirit of Swift if it was something like.

```
public func ifEmpty(default: E) -> Observable<E>
```

and corresponding

```
public func ifEmpty(switch: Observable<E>) -> Observable<E>
```